### ISSUE TITLE
Prefixing selector with div or .table should be allowed

### ISSUE THREAD

**Issue by LHSnow** [flukeout/restaurant#170]
Although not the simplest selector possible, it should be allowed to be more specific than the accepted solution. This is to encourage when a student does it right, even if it is not exactly what was asked for. For example on level 10, where all the following should be correct

```
div *
.table *
*
```

**Comment by loic2665** [comment]
I agree with him, yesterday, our professor gave us this to train CSS Selector, and i was particulary surprised when i typed `.table foo` to select in the table the elements, then few mins later I discovred i don't have to tu `.table` wich is... weird. ^^

**Comment by rogererens** [comment]
Level 10 asks us to 'select all the things', i.e. not just all the things within the div-type (as you select with 'div *'), or within the table-class (as you select with '.table *')
